Studentmediene/kapina-frontend
# Fix/kap 230 fix vulnerabilities

Fixes ~1800 high degree vulnerabilities that were discovered during summer 2019.

To test:
- Delete node_modules locally
- `npm install`
- `npm test` (All tests should pass)
- `PRODUCTION_API=true npm start` and and navigate around the site. Open developer console to make sure no warnings are thrown without you noticing while browsing manually.
